packages/elements/src/tree-select/index.ts

```diff
@@ -239,7 +242,7 @@ export class TreeSelect extends ComboBox<TreeSelectDataItem> {
    * @type {TreeSelectRenderer}
```

Comment:
let's remove this @type as it's duplicate with type info from TypeScript itself.

Reply:
```suggestion
```